libs/langserve/pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[tool.poetry]
+name = "langserve"
+version = "0.0.1"
+description = ""
+readme = "README.md"
+authors = ["LangChain"]
+license = "MIT"
+repository = "https://github.com/langchain-ai/langchain"
+
+[tool.poetry.dependencies]
+python = ">3.8.1,<4"
+httpx = "^0.25.0"
+langchain = { git = "https://github.com/langchain-ai/langchain", subdirectory = "libs/langchain" }
+fastapi = {version = ">0.90.1", optional = true}
+sse-starlette = {version = "^1.6.5", optional = true}
+httpx-sse = {version = "^0.3.1", optional = true}
+pydantic = "^1"
```

Comment:
why pin

Reply:
:shrug: will unpin in next iteration probably after setting up CI to test with both